src/index.js

```diff
@@ -173,6 +181,24 @@ class AudioPlayer extends React.Component {
     }
   }
 
+  addMediaEventListeners (mediaEvents) {
+    if (!mediaEvents) {
+      return;
+    }
+    for (let type in mediaEvents) {
+      this.audio.addEventListener(type, mediaEvents[type]);
```

Comment:
A few quick thoughts:
1. I think we should be checking if `typeof mediaEvents[type] === 'function'` and ignore otherwise - normally React `propTypes` would do this if the handlers weren't nested.
2. If we can use `const` rather than `let`, I prefer it in general.
3. A little nitpicky but I prefer `Object.keys(obj).forEach` to `for ... in`.
4. The above would also apply to `removeMediaEventListeners`.